crates/sui/tests/ptb_files/assign/assign_invalid.ptb

```diff
@@ -0,0 +1,36 @@
+# Raw assign with no value
```

Comment:
I keep forgetting we can add comments to ptb files :))

Reply:
It's obviously very useful to know what we're doing.

Reply:
Indeed it is :) Write (and comment) tests for future you is the way I view it